OrbitLinuxTracing/PerfEvent.cpp

```diff
@@ -36,6 +36,8 @@ void TaskNewtaskPerfEvent::Accept(PerfEventVisitor* visitor) { visitor->visit(th
 
 void TaskRenamePerfEvent::Accept(PerfEventVisitor* visitor) { visitor->visit(this); }
 
+void GenericTracepointPerfEvent::Accept(PerfEventVisitor*) {}
```

Comment:
Why not:
`::Accept(PerfEventVisitor* visitor) { visitor->visit(this); }`

If this is needed to deactivate something for now plese leave a comment with a TODO.